tests/topotests/isis_te_topo1/r1/zebra.conf

```diff
@@ -10,6 +10,7 @@ interface r1-eth0
  link-params
   metric 20
   delay 10000
+  max-bw 1e+10
```

Comment:
Should be set to 10e+10 to be consistent with ospf and cspf test.

Reply:
done